cloudfour/cloudfour.com-patterns
# Chore: style gutenberg table same as c-table elements

## Overview

See #710 

Adds visual front-end support for the following table styles generated via Gutenberg block editor. 

- Default
- Ruled (we get this by default without any additional CSS required)
- Stripped (uses available `is-style-stripes` style option)
- Numeric

## Screenshots

![Screen Shot 2020-05-29 at 2 26 23 PM](https://user-images.githubusercontent.com/1427548/83306971-71970b00-a1b8-11ea-91b1-a415799d23e3.png)

![Screen Shot 2020-05-29 at 2 26 34 PM](https://user-images.githubusercontent.com/1427548/83306972-722fa180-a1b8-11ea-915f-3a00b15c7dce.png)



## Testing

1. View [theme deploy](https://deploy-preview-743--cloudfour-patterns.netlify.app/)
1. Confirm `c-table` component styles are unaffected
1. Observe `vendor/WordPress/gutenberg/core/table` example stories and confirm they appearance match the relevant component styles.
1. Confirm the description/notes for `vendor/WordPress/gutenberg/core/table` is free from typos and written clearly.
